src/models/github-server-app.ts

```diff
@@ -169,6 +181,51 @@ export class GitHubServerApp extends EncryptedModel {
 		});
 	}
 
+	/**
+	 * Update existing GitHubServerApp object
+	 *
+	 * @param {{
+	 * 		gitHubClientId: string,
+	 * 		uuid: string,
+	 * 		appId: number;
+	 * 		gitHubBaseUrl: string,
+	 * 		gitHubClientSecret: string,
+	 * 		webhookSecret: string,
+	 * 		privateKey: string,
+	 * 		gitHubAppName: string,
+	 * 		installationId: number
+	 * 	}} payload
+	 */
+
+	static async updateGitHubAppByUUID(payload: GitHubServerAppPayload): Promise<void> {
+		const {
+			uuid,
+			appId,
+			gitHubAppName,
+			gitHubBaseUrl,
+			gitHubClientId,
+			gitHubClientSecret,
+			webhookSecret,
+			privateKey,
+			installationId
+		} = payload;
+
+		const existApp = await this.findForUuid(uuid);
+		if (existApp) {
```

Comment:
Apparently, you can use the static functions to update a model while still running hooks, but you just specify that option:

```
await Model.update({ username: 'Tony' }, {
  where: { accessLevel: 0 },
  individualHooks: true // this one
});
```

Reply:
And you don't need to check if it exists first, if it's not in the DB, it just won't update anything and continue.  Reducing your DB calls by half.

Reply:
This was something @gxueatlassian because he said there was a bug.

Reply:
Yup I made that changes. I think the `individualHooks` works as well. I am okay either way, but if @mboudreau you insist, @rachellerathbone can you please help me change it? Sorry I need to away from keyboard now.

ps: performance wise, the individual hooks will query from db first. So should be the same.